arch/arm/boot/dts/overlays/sunfounder-pironman5-overlay.dts

```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+    compatible = "brcm,bcm2835";
```

Comment:
We have a lot of overlays that use spaces for indents, including gpio-ir. I'm not bothered about these warnings.